aspnetcore/toc.md

```diff
@@ -33,67 +32,89 @@
 ## [Choose between .NET Core and .NET Framework runtime](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server)
 
 # [MVC](mvc/overview.md)
-## [Create an ASP.NET Core MVC app with Visual Studio](tutorials/first-mvc-app/index.md)
-### [Getting started](tutorials/first-mvc-app/start-mvc.md)
-### [Adding a controller](tutorials/first-mvc-app/adding-controller.md)
-### [Adding a view](tutorials/first-mvc-app/adding-view.md)
-### [Adding a model](tutorials/first-mvc-app/adding-model.md)
-### [Working with SQL Server LocalDB](tutorials/first-mvc-app/working-with-sql.md)
-### [Controller methods and views](tutorials/first-mvc-app/controller-methods-views.md)
-### [Adding Search](tutorials/first-mvc-app/search.md)
-### [Adding a New Field](tutorials/first-mvc-app/new-field.md)
-### [Adding Validation](tutorials/first-mvc-app/validation.md)
-### [Examining the Details and Delete methods](tutorials/first-mvc-app/details.md)
-## [Create a Web API with ASP.NET Core and Visual Studio for Mac](xref:tutorials/first-web-api-mac)
-## [Create a Web API with ASP.NET Core and Visual Studio for Windows](xref:tutorials/first-web-api)
-## [Create a Web API with ASP.NET Core and Visual Studio Code](xref:tutorials/web-api-vsc)
-## [Getting started with ASP.NET Core MVC and Entity Framework Core](data/ef-mvc/index.md)
-### [Getting started](data/ef-mvc/intro.md)
-### [Create, Read, Update, and Delete operations](data/ef-mvc/crud.md)
-### [Sorting, filtering, paging, and grouping](data/ef-mvc/sort-filter-page.md)
-### [Migrations](data/ef-mvc/migrations.md)
-### [Creating a complex data model](data/ef-mvc/complex-data-model.md)
-### [Reading related data](data/ef-mvc/read-related-data.md)
-### [Updating related data](data/ef-mvc/update-related-data.md)
-### [Handling concurrency conflicts](data/ef-mvc/concurrency.md)
-### [Inheritance](data/ef-mvc/inheritance.md)
-### [Advanced topics](data/ef-mvc/advanced.md)
-## [Creating backend services for native mobile applications](mobile/native-mobile-backend.md)
-## [Handling requests with controllers](mvc/controllers/actions.md)
-### [Routing to controller actions](mvc/controllers/routing.md)
-### [Model binding](mvc/models/model-binding.md)
+## [Razor Pages](mvc/razor-pages/index.md)
+## [Models](mvc/models/model-binding.md)
 ### [Model validation](mvc/models/validation.md)
-### [File uploads](mvc/models/file-uploads.md)
-### [Dependency injection into controllers](mvc/controllers/dependency-injection.md)
-### [Testing controllers](mvc/controllers/testing.md)
-## [Rendering HTML with views](mvc/views/overview.md)
+## [Views](mvc/views/overview.md)
 ### [Razor syntax](mvc/views/razor.md)
```

Comment:
Given emphasis now on Razor pages, could this be moved under Razor pages instead of Views?  Or put in both? Or move everything except View compilation from Views to Razor pages?